demo/spec/utils/config_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Shimmer::Config do
+  let(:config) { described_class.instance }
+
+  describe "reading a boolean with the '?' methods" do
+    it "reads '0' as false" do
+      expect(config.shimmer_test_number_zero?).to eq false
```

Comment:
instead of having a fixed `.env` file, you can just set `ENV` variables here which also makes the tests more readable

Reply:
Removed and adapted.